rasa/constants.py

```diff
@@ -59,3 +59,5 @@
 
 DEFAULT_SESSION_EXPIRATION_TIME_IN_MINUTES = 60
 DEFAULT_CARRY_OVER_SLOTS_TO_NEW_SESSION = True
+
+DEFAULT_SENDER_ID = "default"
```

Comment:
@VictorCoCo tests are failing due to a missing newline here

Reply:
Fixed.